pkg/constants/constants.go

```diff
@@ -172,6 +172,14 @@ func RouteName(name string, verb KFServiceVerb) string {
 	return name + "-" + verb.String()
 }
 
+func VirtualServiceName(name string) string {
+	return name + "-vs"
+}
+
+func PredictPrefix(name string) string {
```

Comment:
Wish we'd used the word inference or something better than predict, but I guess we're stuck now.